supportedos.txt

```diff
@@ -0,0 +1,5 @@
+Raspbian=9,10
+Ubuntu=16,18
```

Comment:
You are quite strict on Ubuntu, supporting a two years old version at max? Consider that 20.04 LTS has been released (and their half-year release schedule in general), so I bet you'll face many irritated user reports with this when it leads to an installer exit now, requiring to set an environment variable to pass.

However, matches: https://docs.pi-hole.net/main/prerequisites/
And makes sense that someone tests it thoroughly first, which is probably outstanding, but I wanted to mention.

Reply:
mm, this is a good point.. the actual supported OS is controlled by a TXT record which we can change any time, so it doesn't rely on a release to the repo.

I have not fully tested 20.04 fully, but last time I did there was an issue with uploading debug logs

Reply:
> the actual supported OS is controlled by a TXT record which we can change any time

Jep I recognised that, great Idea btw! This also makes it possible to halt updates in the very first place when a new release drops support for an old OS version that a user might run Pi-hole on currently.